src/main.py

```diff
@@ -1,7 +1,9 @@
 import os
 import importlib
 import pkgutil
-
```

Comment:
Re-add new line here to follow [PEP 8](https://peps.python.org/pep-0008/#imports).